CMakeLists.txt

```
@@ -22,6 +22,9 @@ cmake_minimum_required(VERSION 3.4)
 include(FeatureSummary)
 project(ESPResSo)
 include(cmake/FindPythonModule.cmake)
+if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.12)
```

```suggestion
if(NOT ${CMAKE_VERSION} VERSION_LESS 3.12)
```
version 3.4 has no `VERSION_GREATER_EQUAL` yet